### ISSUE TITLE
React context is not available inside onRenderTab when dragging the tab after updating to 0.6.0

### ISSUE THREAD

**Issue by MariosVisos** [caplin/FlexLayout#285]
I use React's context and more specifically `useSelector` from `react-redux` in a component which I set in `renderValues.content` of the `onRenderTab`. When I drag the tab, I get the following error: `Uncaught Error: could not find react-redux context value; please ensure the component is wrapped in a <Provider>`.

I get this error only in 0.6.0 version.

You can see the problem in the CodeSandbox below.

https://codesandbox.io/s/flexlayout-react-forked-l1shf

**Comment by nealus** [comment]
The problem is that now (in v0.6.0) the drag rectangle tries to render the tab by calling onRenderTab, but the drag rectangle is in a separate top level div not in the react tree under the layout so the context is not available.

You could use the selector in the layout and pass to the handleRenderTab as below:


in app:
```
  import { useSelector } from "react-redux";

  const test = useSelector((state) => state); // get the values using the selector here

  onRenderTab={(tabNode, renderValues) => handleRenderTab(test, tabNode, renderValues)} // pass them to the callback
```

in handleRenderTabs:
```
  const CustomTitleComponent = (props) => {
    // props.state has the values from the selector
    return "Test custom title";
  };

  export const handleRenderTab = (state, tabNode, renderValues) => {
    renderValues.content = <CustomTitleComponent state={state}/>;
  };
```

**Comment by MariosVisos** [comment]
What if I would like to select the state based on a `tabNode`'s value?

**Comment by nealus** [comment]
I've published v0.6.1 with the change

**Comment by MariosVisos** [comment]
It looks good. Thanks a lot for the quick fix. :)
